pkg/virt-api/webhooks/validating-webhook/admitters/vms-admitter.go

```diff
@@ -588,3 +645,80 @@ func validateSnapshotStatus(ar *admissionv1.AdmissionRequest, vm *v1.VirtualMach
 
 	return nil
 }
+
+func (admitter *VMsAdmitter) validateVMUpdate(oldVM, newVM *v1.VirtualMachine) []metav1.StatusCause {
+	if newVM.Status.Ready {
```

Comment:
What's the difference between `newVM.Status.Ready` and `vmi.Status.Phase == Running`?

Reply:
IIUC its same as with k8s pods, when pod is running but its not ready to service requests, it is marked as `0/1` in the `READY` column. The same is here, when `vmi.Status.Phase == Running` is doesn't mean the VMI pod has the ready condition. So `newVM.Status.Ready` means that VMI is running and VMI pod also has the ready condition set to true.

Reply:
If so, I think we should change it to `if vmi.Status.Phase == Running`.
I guess that even if the VM is not ready, but running, it shouldn't be updated.